pagopa/io-app
# chore: [IOBP-833] Sorting algorithms for PSP bundles

## Short description
This pull request updating the sorting algorithm for PSP bundles

* **Updated sorting logic in `getSortedPspList` function**:
 

## List of changes proposed in this pull request
- Updated sorting logic in `getSortedPspList` function

## How to test
- Tap in _Inquadra_ and try to pay something
- Select a payment method
- Check PSP bundles list
- Tap on _Ordina_
- Tap on a radio button to select a sorting option, then verify that the list is sorted correctly according to the chosen algorithm

## Preview

https://github.com/user-attachments/assets/d4af6c03-1843-4bc4-83e2-eecc988c1411
